test-support/page-object/queries.js

```diff
@@ -2,11 +2,23 @@
 
 import { qualifySelector, trim } from './helpers';
 
+function indexedSelector(selector, index) {
+  var iSelector;
```

Comment:
`iSelector` :crying_cat_face:

Maybe we could change `selector` to `baseSelector` and `iSelector` to `selector`?